package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "_s",
+  "version": "1.0.0",
+  "scripts": {
+    "css-build": "npm run css-lint && npm run css-compile && npm run css-prefix && npm run css-format && npm run css-fix",
+    "css-lint": "stylelint style.css",
+    "css-compile": "node-sass sass/style.scss style.css --output-style expanded --indent-type tab --indent-width 1",
+    "css-prefix": "postcss style.css -u autoprefixer -o style.css --no-map",
+    "css-format": "stylefmt style.css",
+    "css-fix": "stylelint style.css --fix",
+    "scss-lint": "stylelint sass/**/*.scss --syntax scss"
```

Comment:
Change this to `stylelint  '**/*.scss' --syntax scss --config node_modules/stylelint-config-wordpress/scss.js`

The glob needs to be quoted and loads the SCSS config from `stylelint-config-wordpress`

Reply:
Updated. Thanks @ntwb !